packages.props

```diff
@@ -6,7 +6,7 @@
 	-->
 	<PropertyGroup>
 		<SystemCollectionsImmutableVersion>6.0.0</SystemCollectionsImmutableVersion>
-		<SystemReflectionMetadataVersion>6.0.0</SystemReflectionMetadataVersion>
+		<SystemReflectionMetadataVersion>6.0.1</SystemReflectionMetadataVersion>
```

Comment:
@siegfriedpammer I take we are intentionally not bumping this beyond 6.0.0 because ICS.D is using it (and thus Roslyn).

Reply:
Yes, please revert this. Thanks!

Reply:
With the exception of ICSharpCode.Decompiler.csproj - what is the point of the exercise anyways?